src/css/custom.css

```
@@ -27,4 +29,33 @@
   --ifm-color-primary-lighter: #32d8b4;
   --ifm-color-primary-lightest: #4fddbf;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.3);
+  --on-premises-color: SteelBlue;
+  --hosted-color: SandyBrown;
 }
+
+.highlight {
+  border-radius: 6px;
+}
+
+.highlight.box {
+  padding: 0.2rem;
+  color: #fff;
+}
+
+.highlight.box.on-premises {
+  background-color: var(--on-premises-color);
+}
+.highlight.box.hosted {
+  background-color: var(--hosted-color);
+}
+
+.highlight.tab.on-premises {
+  border-top: 3px solid var(--on-premises-color);
+}
+.highlight.tab.hosted {
+  border-top: 3px solid var(--hosted-color);
+}
+.highlight.tab.disabled {
+  background-color: lightgray;
```

```suggestion
  color: #bbb;
  background-color: #eee;
```

Currently the grey background makes the tab stand out quite a bit and the lack of toned down foreground color doesn't convey "disabled".